text/0093-remove-shell-processes.md

```diff
@@ -295,7 +232,6 @@ we would convert it to the following in `metadata.toml`
 type = "hi"
 command = ["echo", "hello", "world"]
 args = []
-api = "0.5"
```

Comment:
This ended up not being necessary, as the buildpack-id is actually a part of this table.